fastseq_cli/transformers_generate.py

```diff
@@ -1,5 +1,7 @@
 """From Huggingface Transformers."""
 
+import sys
+import logging
```

Comment:
use the logging wrapped in fastseq as [here](https://github.com/microsoft/fastseq/blob/main/fastseq/optimizer/fairseq/beam_search_optimizer.py#L24)?